src/bors/command/parser.rs

```diff
@@ -242,6 +287,38 @@ mod tests {
         assert!(matches!(cmds[0], Err(CommandParseError::DuplicateArg("a"))));
     }
 
+    #[test]
+    fn parse_default_approve() {
+        let cmds = parse_commands("@bors r+");
+        assert_eq!(cmds.len(), 1);
+        assert!(matches!(
+            cmds[0],
+            Ok(BorsCommand::Approve(Approver::Myself))
+        ));
+    }
+
+    #[test]
+    fn parse_approve_on_behalf() {
```

Comment:
Maybe add another test for `bors r=foo,bar`, people sometimes use multiple names here.